test/libsolidity/ASTPropertyTest.cpp

```diff
@@ -0,0 +1,175 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+// SPDX-License-Identifier: GPL-3.0
+
+#include <test/libsolidity/ASTPropertyTest.h>
+#include <test/Common.h>
+
+#include <libsolidity/ast/ASTJsonExporter.h>
+#include <libsolidity/interface/CompilerStack.h>
+
+#include <liblangutil/Common.h>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/throw_exception.hpp>
+
+#include <range/v3/range/conversion.hpp>
+#include <range/v3/view/split.hpp>
+
+#include <queue>
+
+using namespace solidity::langutil;
+using namespace solidity::frontend;
+using namespace solidity::frontend::test;
+using namespace solidity;
+using namespace std;
```

Comment:
We could already remove the `std` from here as well.

Reply:
Please no, and thanks for asking 😅.